content/applications/productivity/discuss/overview/plan_activities.rst

```diff
@@ -1,64 +1,192 @@
+   are currently installed in the database.
+
+Default user
+~~~~~~~~~~~~
+
+To automatically assign this activity to a specific user when this activity type is scheduled,
+choose a name from the :guilabel:`Default User` drop-down. If this field is left blank, the activity
+will be assigned to the user who creates the activity.
+
+Default summary
+~~~~~~~~~~~~~~~
+
+To include any notes every time this activity type is created, enter them into the
+:guilabel:`Default Summary` field.
+
+.. note::
+   The information in the :guilabel:`Default User` and :guilabel:`Default Summary` fields will be
+   included when an activity is created. However, they can be altered before the activity is
+   scheduled or saved.
 
-Odoo helps you to plan a flow of activities by allowing you to set *Recommended Next Activities*.
+Next activity
+-------------
 
-.. image:: plan_activities/recommended_activities.png
+To automatically suggest or trigger a new activity after an activity has been marked complete, the
+*Chaining Type* must be set.
+
+Suggest next activity
+~~~~~~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Chaining Type` field, select :guilabel:`Suggest Next Activity`. The field
+underneath will now be titled :guilabel:`Suggest`. Select any activities to recommend as follow-up
+tasks to this activity type.
+
+In the :guilabel:`Schedule` field, choose a default deadline for these activities. This can be
+altered before the activity is scheduled.
+
+Click :guilabel:`Save`.
+
+.. figure:: plan_activities/schedule-recommended-activity.png
    :align: center
-   :alt: View of an activity type form emphasizing the field recommended next activities for Odoo
-         Discuss
 
-Once the respective activity is completed, select *Done & Schedule Next* and next steps are
-suggested to you.
+   If an activity has the chaining type set to suggest, and has activities listed in the suggest
+   field, users are presented with recommendations for activities as next steps.
+
+Trigger next activity
+~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: plan_activities/schedule_recommended_activity.png
+Setting the *chaining type* to *Trigger* immediately launches the next activity once the previous
+one is completed.
+
+In the :guilabel:`Chaining Type` field, select :guilabel:`Trigger Next Activity`. The field
+underneath will now be titled :guilabel:`Trigger`. Select the activity that should be launched once
+this activity is completed.
+
+In the :guilabel:`Schedule` field, choose a default deadline for these activities. This can be
+altered before the activity is scheduled.
+
+Click :guilabel:`Save`.
+
+.. figure:: plan_activities/triggered-activities.png
    :align: center
-   :alt: View of an activity being schedule emphasizing the recommended activities field being
-         shown for Odoo Discuss
+
+   When an activity has the chaining type set to trigger, marking the activity as Done immediately
```

Comment:
```suggestion
   :alt: When an activity has the chaining type set to trigger, marking the activity as Done immediately
```

Reply:
This sentence could be before the image, that way the image isn't out of place. Modify alt tag

Reply:
Not an alt tag, its the caption